cms/jinja2/templates/pages/statistical_article_page--previous-releases.html

```diff
@@ -1,28 +1,80 @@
 {% extends "templates/base_page.html" %}
 
+{% from "components/document-list/_macro.njk" import onsDocumentList %}
```

Comment:
suggestion: also update the `page_title` (used in the HTML `<title>`

{%- set page_title -%}
{% trans trimmed title=page.title %}
Previous releases for {{ title }}
{% endtrans %}
{%- endset -%}

Reply:
this last bit needs [base-html-title-tweak.patch](https://github.com/user-attachments/files/20366407/base-html-title-tweak.patch)
for it to work

Reply:
Nice one thanks @zerolab ! Updated :)